apps/extension/webpack/webpack.common.js

```diff
@@ -70,6 +70,7 @@ module.exports = {
   },
   plugins: [
     new webpack.ProvidePlugin({
+      // Required by the `bip39` library
```

Comment:
Added this note since I originally removed `buffer` from the extension, thinking we weren't using it anywhere.